lib/bridge/MatrixHandler.js

```diff
@@ -451,6 +451,27 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
         yield this.ircBridge.sendMatrixAction(adminRoom, botUser, notice, req);
         return;
     }
+    else if (cmd === "!storepass") {
+        let domain = ircServer.domain;
+        let userId = event.user_id;
+        let notice;
+
+        try {
+            let force = args[1] === '-f';
+            yield this.ircBridge.getStore().storePass(userId, domain, args[0], force);
```

Comment:
Right now I see why you mentioned `-f` before. Why is `-f` A Thing? This should not be an option for random Matrix users.